Makefile

```diff
@@ -1,11 +1,14 @@
+.DEFAULT_GOAL = test
+
 lint:
-	luacheck lua/lualine
-	luacheck tests
+	@luacheck lua/lualine
+	@luacheck tests
 
 format:
-	for file in `find -name '*.lua'`;do lua-format $$file -i; done;
+	@for file in `find -name '*.lua'`;do lua-format $$file -i; done;
 
 test:
```

Comment:
@hoob3rt is there any difference in making this line?
```
    test: lint
```
instead of @make lint in following line ?

Reply:
> @hoob3rt is there any difference in making this line?
> 
> ```
>     test: lint
> ```
> 
> instead of @make lint in following line ?

yes there is, your solution is better :smile: